docs/ref/models/fields.txt

```diff
@@ -502,6 +502,15 @@ Any combination of these options will result in an error.
     ``True`` will cause the field to have ``editable=False`` and ``blank=True``
     set.
 
+.. note::
+    When using timezones, storing a ``DateField`` in the database is
+    ambiguous because it's incapable of storing timezone aware information, unlike ``DateTimeField``.
+    Unless you know you will need ``DateField``, consider using ``DateTimeField`` instead
+    and convert to a date format for display only.
```

Comment:
It wasn't clear to me when reviewing the PR that this note was scoped within the documentation of `auto_now(_add)`. Sorry about that.

In my opinion, it's easier to replace a `DateField` with `auto_now(_add)` by a `DateField` with:

    def save(self, *args, **kwargs):
        today = datetime.date.today()  # which depends on settings.TIME_ZONE ;-)
        if self.pk is None:
            self.created_at = today
        self.updated_at = today
        super().save(*args, **kwargs)

than by a `DateTimeField` with `auto_now(_add)`. At least you see what's going on.

Now, as long as your website operates in a single time zone, which is true for most websites (even if USE_TZ is set) none of this is an issue... So let's make sure we don't create complications where there aren't any...

I think we should document that `auto_now` and `auto_now_add` operate in the default time zone and suggest to override `save()` if this this is a problem -- that is, if the current time zone can be different from the default time zone.